packages/host/app/components/operator-mode/code-submode/playground/playground-panel.gts

```diff
@@ -102,12 +102,14 @@ export default class PlaygroundPanel extends Component<Signature> {
 
   @tracked private cardResource: ReturnType<getCard> | undefined;
   @tracked private fieldChooserIsOpen = false;
+  @tracked private cardCreationError: CardErrorJSONAPI | undefined = undefined;
 
   private get moduleId() {
     return internalKeyFor(this.args.codeRef, undefined);
   }
 
   private get isLoading() {
+    this.clearCardCreationError();
```

Comment:
Normally, it is not good practice to have a side effect on a getter. Is there a root source of state that we are missing that is pushing us in this direction?

Reply:
Maybe there's a reactive way to handle this in the store for the failing card-creation case? @habdelra